src/vcpkg/binarycaching.cpp

```diff
@@ -746,6 +746,147 @@ namespace
         bool m_interactive;
         bool m_use_nuget_cache;
     };
+
+    bool gsutil_stat(const std::string& url)
+    {
+        System::Command cmd;
+        cmd.string_arg("gsutil").string_arg("-q").string_arg("stat").string_arg(url);
```

Comment:
We may want to add `gsutil` to https://github.com/microsoft/vcpkg-tool/blob/main/src/vcpkg/tools.cpp to eventually support automatic download via `vcpkgTools.xml` (https://github.com/microsoft/vcpkg/blob/master/scripts/vcpkgTools.xml).

Reply:
That does seem like a good idea.  I doubt I will have time to do this until the weekend. We can wait or merge now, your call, but I prefer to merge sooner rather than later.